src/components/Articles.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { fetchArticles } from "../api";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+//   const [isLoading, setIsLoading] = true;
+
+  useEffect(() => {
+    fetchArticles().then((articleData) => {
+      setArticles(articleData);
+      console.log(articleData)
+      console.log(articles)
+      
+    //   setIsLoading(false);
+    });
+  }, []);
+
+//   if (isLoading) {
+//     return <div>Loading...</div>;
+//   }
+
+  return(
+  <div className="article-list">
+{articles.map((articles)=>(
+    <div className="article" key={articles.article_id}>
```

Comment:
You could make use of some more semantic html here by using <ul> and <li> if you want